src/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -314,37 +293,24 @@ private void resetDefaultGame() {
   }
 
   public void loadDefaultGame(final Component ui) {
-    // clear out dynamix's properties (this ended up being the best place to put it, as we have definitely left a game at this point)
+    // clear out dynamix's properties (this ended up being the best place to put it, as we have
+    // definitely left a game at this point)
     Dynamix_AI.gameOverClearCachedGameDataAll();
     ProAI.gameOverClearCache();
-    loadDefaultGame(ui, false);
-  }
 
-  /**
-   * @param ui
-   * @param forceFactoryDefault
-   *        - False is default behavior and causes the new game chooser model to be cleared (and refreshed if needed).
-   *        True causes the default game preference to be reset, but the model does not get cleared/refreshed (because we only call with
-   *        'true' if loading the user preferred map failed).
-   */
-  private void loadDefaultGame(final Component ui, final boolean forceFactoryDefault) {
-    // load the previously saved value
     final Preferences prefs = Preferences.userNodeForPackage(this.getClass());
-    if (forceFactoryDefault) {
-      // we don't refresh the game chooser model because we have just removed a bad map from it
-      resetDefaultGame();
-    }
+    final String userPreferredDefaultGameURI = prefs.get(DEFAULT_GAME_URI_PREF, DEFAULT_GAME_URI);
     NewGameChooserEntry selectedGame = null;
```

Comment:
Paths of code where forceFactoryDefault is false are removed. This method was changed from loadDefaultGame(Component , boolean ) to be instead loadDefaultGame( Component) and the boolean parameter is always false (and so is removed). Clients that called this method before with a "true" value now instead call loadFactoryDefaultGame. There you should see "resetDefaultGame()" game code.